styles/styles.css

```diff
@@ -246,20 +246,90 @@ body::-webkit-scrollbar-thumb{
 }
 
 /* tablet size */
-@media only screen and (max-width: 800px){
+@media only screen and (max-width: 50rem){
     .nav-bar-container{
         grid-template-columns: 1fr 1fr 1fr;
         width: 100%;
     }
     .logo{
        display: none;
    }
+
+   .bio-container{
+        grid-template-columns: 1fr;
+        margin: auto;
+   }
+
+   .work-container{
+       grid-template-columns: 1fr 1fr;
+   }
 }
 
 /* mobile size */
-@media only screen and (max-width: 440px) {
-    .nav-bar-container {
-        grid-template-columns: 1fr;
+@media only screen and (max-width: 28rem) {
+    .nav-bar-container, .nav-cross {
```

Comment:
Keep this consistent, you have a space after the selectors here. Either remove the space here or add it elsewhere (bit of a pain, but having a space there is considered to be standard)